docs/getting-started.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
More work needs to be done on this for sure, but I was kind of mapping some things out here

Reply:
I don't know if this is "out of scope" here, but I think it would be preferable to have a "Getting started" section rather than a page. Then each "step" like deploy Cumulus, create rules/collections, would be separate pages in this section. And it would fully walk you through deploying to setting up your first workflow.

My concern about the "one page" solution is that I think linking out to other random pages throughout the docs for things like creating rules is a bit confusing, just because the information feels disorganized from where you actually need to use it. It would be nice if the "Getting started" section just had all the information you need to get started within the pages in that section itself. This is just my opinion though.

Reply:
But then, it shouldn't have all of the info for deployments, right? Like deployments would be its own section. So maybe there is an operator getting started section? but then overall you'd want a page that tells you to go to deployments first, then use the dashboard?

Reply:
I took it off the sidebars for now though.

Reply:
Personally, I would just move the deployment README into the "getting started" section. I envision the pages being something like:

- Intro to cumulus
- Intro to cumulus "concepts": rules/collections/providers
- Deploying cumulus
- Writing your first workflow
- Deploying/setting up the dashboard

Then other info related to deployment like troubleshooting or additional config options would be in another section. But this is just my opinion